css/style.css

```diff
@@ -1,7 +1,388 @@
+}
+
+.vis h1 {
+  color: var(--primary-text-color);
+}
+
+.vis h2 {
+  color: var(--secondary-text-color);
+}
+
+.info-panel-controls {
+  display: flex;
+  flex-direction: column;
+  margin-right: -30px;
+  margin-top: -30px;
+}
+
+.website {
+  background-color: transparent;
+  border-radius: 3px 0 0 3px;
+  margin-top: 0;
+  box-shadow: none;
+}
+
+.website::before {
+  background-image: url('../images/lightbeam_icon_website.png');
+  margin-right: 0;
+  opacity: .5;
+}
+
+.help {
+  border-radius: 3px 0 0 3px;
+  margin-top: 5px;
+}
+
+.help::before {
+  background-image: url('../images/lightbeam_icon_help.png');
+  margin-right: 0;
+}
+
+.about {
+  border-radius: 3px 0 0 3px;
+  margin-top: 5px;
+}
+
+.about::before {
+  background-image: url('../images/lightbeam_icon_about.png');
+  margin-right: 0;
+}
+
+/* ----------- UI - vis Controls ----------- */
+
+footer {
+  grid-row-start: 2;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr) 150px;
+}
+
+footer h2 {
+  padding-bottom: 5px;
+  border-bottom: 1px solid var(--primary-text-color);
+}
+
+footer div:first-child {
+  grid-column-end: span 4;
+}
+
+.controls {
+  display: grid;
+  grid-column-gap: 10px;
+  grid-template-columns: repeat(3, 150px);
+  grid-template-rows: repeat(3, 1fr);
+}
+
+.visited-sites,
+.third-party-sites,
+.connections {
+  grid-column-start: 1;
+}
+
+.visited-sites::before {
+  background-image: url('data:image/svg+xml,<svg style="fill: rgb(234, 234, 234)" version="1.1" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="6"></circle></svg>');
+}
+
+.third-party-sites::before {
+  background-image: url('data:image/svg+xml,<svg style="fill: rgb(234, 234, 234)" version="1.1" xmlns="http://www.w3.org/2000/svg"><polygon points="0,14 7,2 14,14"></polygon></svg>');
+}
+
+.connections::before {
+  background-image: url('data:image/svg+xml,<svg version="1.1" xmlns="http://www.w3.org/2000/svg"><line x1="14" y1="2" x2="2" y2="14" style="stroke: rgb(234, 234, 234); stroke-width: 2"></line></svg>');
+}
+
+.watched-sites {
+  grid-column-start: 2;
+  grid-row-start: 1;
+
+}
+
+.watched-sites::before {
+  background-image: url('data:image/svg+xml,<svg style="fill: rgb(111, 195, 229)" version="1.1" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="6"></circle></svg>'), url('data:image/svg+xml,<svg style="margin-left: 15px; fill: rgb(111, 195, 229)" version="1.1" xmlns="http://www.w3.org/2000/svg"><polygon points="0,14 7,2 14,14"></polygon></svg>');
+  width: 30px;
+}
+
+.blocked-sites {
+  grid-column-start: 2;
+  grid-row-start: 2;
+}
+
+.blocked-sites::before {
+  background-image: url('data:image/svg+xml,<svg style="fill: rgb(224, 42, 97)" version="1.1" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="6"></circle></svg>'), url('data:image/svg+xml,<svg style="margin-left: 15px; fill: rgb(224, 42, 97)" version="1.1" xmlns="http://www.w3.org/2000/svg"><polygon points="0,14 7,2 14,14"></polygon></svg>');
+  width: 30px;
```

Comment:
Nit: What is distinct about `.blocked-sites` and `.watched-sites` that require `width: 30px;` that `.connections` doesn't? Can they share a selector to do this?

```
.bargh > button::before {
  width: 30px;
}
```

I haven't looked at the layout so not sure, however if the icon itself shouldn't really be tied to width, margin etc.